src/main.rs

```diff
@@ -68,7 +68,13 @@ fn main() {
             if let Some(e) = err.downcast_ref::<edgedb_errors::Error>() {
                 print::edgedb_error(e, false);
             } else {
-                print::error(err);
+                let mut error_chain = err.chain();
+                if let Some(first) = error_chain.next() {
+                    print::error(first);
+                }
```

Comment:
Do we expect this to always execute?

Reply:
Yes, it would be very strange if the error would have an empty chain. Although it can happen, maybe, due to a bug upstream or something else I don't know about.

If that happens it would be better if we print something, though.